__tests__/app.test.js

```diff
@@ -105,6 +105,25 @@ describe("/api/articles", ()=>{
       expect(body.articles).toBeSortedBy("created_at", { descending: true });
     });
   });
+  test("200: responds with array of articles filtered by the topic mitch", ()=>{
+    return request(app)
+    .get("/api/articles?topic=mitch")
+    .expect(200)
+    .then(({ body })=>{
+      expect(body.articles[0].topic).toBe("mitch");
+      expect(Array.isArray(body.articles)).toBe(true);
+    })
+  })
+  test("400: should return 400 status when given an invalid topic", () => {
+    return request(app)
+    .get("/api/articles?topic=1234")
+    .expect(404);
+  })
+  test("404: should return 404 status when given a non existing topic", () => {
+    return request(app)
+    .get("/api/articles?topic=doesnotexist")
+    .expect(404);
```

Comment:
This test is therefore a duplicate of the one above, however another scenario you could test for is if the topic is _found_ but has no _articles_